source/style-guide/seo-guidelines.txt

```diff
@@ -0,0 +1,84 @@
+.. _seo-guidelines:
+
+=====================================
+Search Engine Optimization Guidelines
+=====================================
+
+.. default-domain:: mongodb
+
+Search Engine Optimization (SEO) considers the actual terms that users 
+enter into serach engines (keywords) and employs best practices to 
+improve traffic to web pages from search engines. The following 
+guidance should help you optimize your documentation pages for search.
+
+Titles
+------
+
+Title pages and subsections according to the following SEO best 
+practices:
+
+- Use a maximum of 70 characters.
+
+- Include target keywords.
+
+- Avoid excessive or irrelevant words (keyword stuffing).
+
+- Use a unique title for every page.
+  
+To learn more, see :ref:`titles-and-headings`.
+
+Alternative Text
+----------------
+
+Screen readers read alternative text aloud so that users can better
+understand an on-page image. Specify alterative text according to the 
+following SEO best practices:
+
+- Use a maximum of 125 characters per image.
+
+- Describe the image with specific language.
+
+- If it makes sense, include keywords.
+
+To learn more, see :ref:`write-for-accessibility`.
+
+Keywords
+--------
+
+You can add the actual terms that users enter into search engines 
+(keywords) to your page to improve its SEO. Add keywords according to 
+the following best practices:
+
+- Take the most concise form of the information that the page conveys 
+  and make that the target keyword.
+
+- Don't add keywords so broad that they compete with the product page.
+
+- Don't add keywords so specific that we miss the actual behavior of 
+  our searchers. For example, add “mongodb atlas course” instead of 
+  “Getting Started with Atlas Data Platform”.
+
+To learn more, see :ref:`metadata`.
+
+Descriptions
+------------
+
+The snippet that appears under the link in the search results can 
+impact SEO. Write these descriptions according to the following best 
+practices:
+
+- Make the descriptions informative (and enticing if possible).
+
+- Emphasize the “why” for using the page.
+
+- Use a maximum of 155 characters.
+
+- Include target keywords and a call to action.
+
+- Keep the description short, simple, and concise. 
+
+- Concisely, explain the page content.
+
+- Use unique meta descriptions for every page.
```

Comment:
Suggestion:

I think it's probably better to stay consistent in the terminology ("descriptions" vs "meta descriptions"). It currently seems like "descriptions" is the term used more often, so it could be good to use that here too:

```suggestion
- Use unique descriptions for every page.
```